order.go

```diff
@@ -118,10 +118,10 @@ type OrderListParams struct {
 // StatsuTransitions are the timestamps at which the order status was updated
 // https://stripe.com/docs/api#order_object
 type StatusTransitions struct {
-	Canceled  int64 `json:"canceled"`
-	Fulfilled int64 `json:"fulfiled"`
-	Paid      int64 `json:"paid"`
-	Returned  int64 `json:"returned"`
+	Canceled int64 `json:"canceled"`
+	Fulfiled int64 `json:"fulfiled"`
```

Comment:
I think we should leave this as `Fulfilled` and fix it server side at some point instead.

No point in cargo culting too hard.